e3db/types/proxy_authorizer_policy.py

```diff
@@ -0,0 +1,156 @@
+import uuid
+
+
+class ProxyAuthorizerPolicy(object):
```

Comment:
After talking with @m4dc4p about this, I think the best way forward would actually be to unify on the response returned from `/granted` and `/proxies`. The single field difference (`*_name`) is actually not very useful at this point, so I think it could be worth ignoring in the SDKs and possibly even removing from the server response. I'll consider updating the spec and server. I appreciate your work here though!